source/containers.tex

```diff
@@ -2522,7 +2522,7 @@
 &   \tcode{void}
 &   \requires\ \tcode{value_type} shall be \tcode{EmplaceConstructible} into \tcode{X} from \tcode{*i}.\br
     \requires \tcode{i} and \tcode{j} are not iterators in \tcode{a}.
-    Equivalent to \tcode{a.insert(t)} for each element in \tcode{[i,j)}.%
+    \effects\ Equivalent to \tcode{a.insert(t)} for each element in \tcode{[i,j)}.%
     \indextext{unordered associative containers!\idxcode{insert}}%
```

Comment:
I'd prefer leaving container requirements tables alone; they need a serious overhaul anyway, and it's unclear whether \effects and friends should or should not appear there.

Reply:
That's true, but for the time being, I think this change improves local consistency, so I don't mind it.